app/assets/javascripts/scp-related-genes-ideogram.js

```diff
@@ -76,18 +76,39 @@ function showRelatedGenesIdeogram() { // eslint-disable-line
   ideoContainer.classList = 'show-related-genes-ideogram'
 }
 
-/**
- * Get summary of related-genes ideogram that was just loaded or clicked
- */
-function getRelatedGenesAnalytics(ideogram) {
-  const props = Object.assign({}, ideogram.relatedGenesAnalytics)
-
+/** Refine analytics to use DSP-conventional names */
+function conformAnalytics(props, ideogram) {
   // Use DSP-conventional name
-  props['perfTime'] = props.time
-  delete props['time']
+  props['perfTime'] = props.timeTotal
+  delete props.timeTotal
+
+  props['species'] = ideogram.organismScientificName
+
+  return props
+}
 
-  props['species'] = ideogram.getScientificName(ideogram.config.taxid)
+/** Log hover over related genes ideogram tooltip */
+function onWillShowAnnotTooltip(annot) {
+  // Ideogram object; used to inspect ideogram state
+  const ideogram = this // eslint-disable-line
+  let props = ideogram.getTooltipAnalytics(annot)
+
+  // `props` is null if it is merely analytics noise.
+  // Accounts for quick moves from label to annot, or away then immediately
+  // back to same annot.  Such action flickers tooltip and represents a
+  // technical artifact that is not worth analyzing.
```

Comment:
would a good long-term (outside the scope of this PR) be to change the event to 'didShowTooltip' so that the artifacts can get filtered out before this is called?

Reply:
I like the gist!  I'll try incorporating something like that upstream, so clients don't need to filter out this noise.